internal/controller/controllers/agent_controller.go

```diff
@@ -588,11 +594,31 @@ func (r *AgentReconciler) updateStatus(ctx context.Context, log logrus.FieldLogg
 	} else {
 		setConditionsUnknown(agent)
 	}
+	// get node
+	var node *corev1.Node
+	if funk.Contains([]models.HostStage{models.HostStageRebooting, models.HostStageJoined, models.HostStageConfiguring}, agent.Status.Progress.CurrentStage) {
```

Comment:
1. Is it also relevant to day1? According to the code it seems it does. Day1 flow shouldn't be handled here.
2. For day2 flow, seems that only none platform is handled.  Is this the intention?